src/diffcalc_API/controllers/HklCalculation.py

```diff
@@ -0,0 +1,106 @@
+from itertools import product
+from typing import Dict, List, Tuple
+
+import numpy as np
+from diffcalc.hkl.calc import HklCalculation
+from diffcalc.hkl.geometry import Position
+
+from diffcalc_API.errors.HklCalculation import (
+    check_valid_miller_indices,
+    check_valid_scan_bounds,
+)
+
+positionType = Tuple[float, float, float]
```

Comment:
This is a type so should start with a capital

```suggestion
PositionType = Tuple[float, float, float]
```